Source/Fuse.Nodes/Visual.BeginRemove.uno

```diff
@@ -84,7 +84,7 @@ namespace Fuse
 			}
 			else
 			{
-				args.Remove();				
+				UpdateManager.AddDeferredAction(args.Remove);
```

Comment:
This is probably going to require that a lot more tests do root.PumpDeferred(), which makes me skeptical to this change after testing has been performed.